### DISCUSSION TITLE
How to set kernel configs in 24.04?

### DISCUSSION THREAD

**Discussion by peyton-howe** [Joshua-Riek/ubuntu-rockchip#762] [Q&A] [answered]
Which file do you modify to set the kernel configs? I assumed it was `linux_rockchip_defconfig` like in 22, but this didn't appear to be the case when I built my kernel.

I set the following two lines in the config to enable the raspberry pi V1 and V2 camera drivers to be built-in modules.
```
CONFIG_VIDEO_OV5647=y
CONFIG_VIDEO_IMX219=y
```

And then I compiled the kernel like so.
```
export CC=aarch64-linux-gnu-gcc
export $(dpkg-architecture -aarm64)
export CROSS_COMPILE=aarch64-linux-gnu-

LANG=C fakeroot debian/rules clean binary-rockchip
```

After compiling, I installed all of the built .deb files using dpkg and rebooted. Upon reboot, modinfo shows that both the OV5647 and IMX219 camera drivers are loadable modules instead of built-in. What am I missing here?

**Comment by Joshua-Riek** [comment, marked as answer]
I need to update docs for the kernel, but you need to modify the config in debian.rockchip/config/config.common.ubuntu

**Comment by Ruach** [comment]
@peytonicmaster6 Hi I am curious how could you load the custom built linux image to the sd card.
When you say dpkg built .deb files, does it mean that you compile the linux on rk3588 processor and then dpkg built.deb files to replace kernel?
